src/libaktualizr/uptane/tuf.h

```diff
@@ -262,6 +262,10 @@ class Target {
   void setUri(std::string uri) { uri_ = std::move(uri); };
   bool MatchHash(const Hash &hash) const;
 
+  void InsertEcu(const EcuSerial &ecuIdentifier, const HardwareIdentifier &hwid) {
```

Comment:
Why not 'insert' the pair? It's declared as the pair in the lite client context `std::pair<Uptane::EcuSerial, Uptane::HardwareIdentifier> primary_ecu;` so passing it around as the pair would be a bit more convenient, IMHO.

Reply:
Good point.  Fixed